package NYT.NTesting;

message TRow {
    optional string string_field = 1;
    optional int32 int32_field = 2;
    optional fixed64 fixed64_field = 3;
}
